server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsResponse.java

```diff
@@ -145,6 +186,12 @@ public XContentBuilder toXContent(XContentBuilder builder, ToXContent.Params par
         if (next != null) {
             builder.field("next", next);
         }
+        if (totalCount >= 0) {
```

Comment:
I think we will remove these conditions in master after backport and release of 7.15 since then it will always be present in master? Just a clarification.

Reply:
Right these are just for BwC.